### ISSUE TITLE
Update paperless-ng to latest release

### ISSUE THREAD

**Issue by petewasher** [TheBestMoshe/home-assistant-addons#1]
Hello, 
Firstly thanks for creating this addon - it came at just the right time for me! It has been working great for me so far. 

Are you going to update to the latest version of `paperless-ng`? I see that currently you're using `0.9.13` which is labelled as 'pre-release', but `1.1.0` is available and marked as a as 'Release' which has some useful looking [changes](https://paperless-ng.readthedocs.io/en/latest/changelog.html). 

Do you know if there is any awkward migration path for upgrading?

Thanks again.

**Comment by TheBestMoshe** [comment]
@petewasher Sorry I missed your message. 

Are you still interested in collaborating on this? I see you've created your own fork of the project. 

If you prefer to maintain a fork, that works as well.

**Comment by petewasher** [comment]
Hello, yes, I am interested. Makes sense to keep things in one place. 

Since I was getting to know home assistant add-ons anyway, I've created and pushed the docker images [here](https://hub.docker.com/search?q=cyclickdevelopment%2Fpaperless&type=image). I used the official home-assistant builder to create them.

There were quite a few changes to make to get hold of the latest requirements etc (some of which were solved by using Paperless-ng `1.1.4` instead of `1.1.0` as I originally suggested). 

On my fork the only changes needed were in the [Dockerfile](https://github.com/petewasher/home-assistant-addons/commits/master/paperless-ng/Dockerfile). Looks like some things moved in the paperless-ng repository, and I fixed a couple of warnings with `pip`.

I've raised a PR with the necessary changes.
